runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientFetchFactory.java

```diff
@@ -1963,6 +1963,7 @@ private void doApplicationFlush(
             int reserved,
             Flyweight extension)
         {
+            flushOrDataFramesSent++;
```

Comment:
Please move this to the end of the method, so it represents what has already happened.